test/mock/collector/nginx-agent.conf

```diff
@@ -55,6 +55,7 @@ collector:
           key: /tmp/key.pem
           generate_self_signed_cert: true
   processors:
+    attribute: {}
```

Comment:
Dont think you need to specify an empty attribute in the config since you are programmatically adding it